src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceApiServiceSpec.scala

```diff
@@ -486,6 +489,15 @@ class WorkspaceApiServiceSpec extends BaseServiceSpec with WorkspaceApiService w
         }
       }
     }
+
+    "Passthrough tests on the /workspaces/genomics/operations/%s path" - {
+      "OK status is returned for GET" in {
+        stubRawlsService(HttpMethods.GET, genomicsOperationsPath, OK)
+        Get(genomicsOperationsPath) ~> dummyUserIdHeaders("1234") ~> sealRoute(workspaceRoutes) ~> check {
+          status should equal (OK)
+        }
+      }
+    }
```

Comment:
Suggestion: For pass-throughs, we usually have tests to ensure we're NOT handling the http methods we do not want to pass-through, i.e. PUT, POST, etc., and expect the right method-not-handled error.